doc/ref_model/chapters/chapter10.md

```diff
@@ -43,8 +43,12 @@ This section addresses major open issues identified in the development of the Re
 ## 10.3.1 Discovery [Initial language]
 The VNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. CNTT reference model specifies a model, but OpenStack does not provide APIs to support such capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
 
-## 10.3.2 Support Load Balance of VNF/CNFs [Initial language]
-Ability to load balance workflows through multiple instances of same VNF or CNF - e.g. using ECMP to distribute workloads through the multiple instances of Firewall. As an example imagine a distributed finance application with multiple instances of Web-tier and DB-tier. The traffic needs to flow through multiple instances of Firewall (for HA as well load balancing). There is no simple way to accomplish this in OpenStack.
+## 10.3.2 Support Load Balance of VNF/CNFs
+The ability to dynaically scale a network function by load balancing accross multiple instances/replicas of the same VNF or CNF is essential. New architectures and application patterns such as micro services is making this even more crucial. It must not only be possible to load balance and scale each service layer independently, support to chain the different layers together through "Service Funtion Chaining" is also needed. 
+
+The load balancing and scaling need for typical enterprise applications is well supported in Openstack by the LBaaSv2 API and Octavia, the built in mechanism in Kubernetes for scaling enterprise type of services and PODs is also sufficient for applications that only use one interface.
```

Comment:
Openstack -> OpenStack
Kubernetes does _not_ have load balancing capabilities between applications.

Reply:
I am not sure I understand your comment.
Openstack has an LB API, no problem there, right?
For kubernetes you have the Service of type loadbalancer spec for external use and the standard Service spec for internal load balancing of PODs (https://kubernetes.io/docs/concepts/services-networking/service/). So, there is load balancing between applications in K8s. The difference is that you can not create a vLB and then provision it. The LB provisioning is done by creating a service spec. This is of course only available on the primary network interface (which is the only interface that is a K8s support any type of services on). There is little to none support for any NW functionality on the secondary interfaces from a K8s standpoint, all this is done by CNI plugins like Multus, which today support NW attachment, IPAM and interface specific routes